contrib/test_jenkins.sh

```diff
@@ -1488,70 +1488,11 @@ run_tests() {
 	export UCX_ERROR_MAIL_TO=$ghprbActualCommitAuthorEmail
 	export UCX_ERROR_MAIL_FOOTER=$JOB_URL/$BUILD_NUMBER/console
 
-	# test cuda build if cuda modules available
```

Comment:
seems we are losing "some" test coverage?

Reply:
oooh, forgot to remove that